client/state/data-layer/wpcom-http/utils.js

```diff
@@ -72,7 +72,7 @@ export const makeParser = ( schema, schemaOptions = {}, transformer = identity )
 	let validate;
 
 	const genParser = () => {
-		const options = Object.assign( { verbose: true }, schemaOptions );
+		const options = Object.assign( { greedy: true, verbose: true }, schemaOptions );
```

Comment:
Should we set `greedy: 'development' === process.env.NODE_ENV` like what's below? That should help keep other environments as performant as possible.

Reply:
maybe but I think `greedy` might be useful in all cases as it can open up better errors.

the question it hinges on is really whether we're tracing these errors centrally or not.
this performance shouldn't matter because it's only happening on network response and should usually only operate in these cases where the bottleneck is magnitudes slower than the parser

Reply:
👍 fair enough